package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "start": "node server",
-    "dev": "cross-env NODE_ENV=dev node server",
+    "dev": "concurrently --kill-others \"cross-env NODE_ENV=dev node server\"",
     "release": "cross-env NODE_ENV=production webpack --config ./config/webpack.config.release.js && cp -r server build/release && cp package.json build/release && cd build/release && npm install --only=prod --ignore-scripts",
```

Comment:
```
$ pm run release 

> starter@0.1.0 release
> cross-env NODE_ENV=production webpack --config ./config/webpack.config.release.js && cp -r server build/release && cp package.json build/release && cd build/release && npm install --only=prod --ignore-scripts

[webpack-cli] Failed to load '.../obsidianjs-starter/config/webpack.config.release.js' config
[webpack-cli] Invalid options object. Terser Plugin has been initialized using an options object that does not match the API schema.
 - options has an unknown property 'sourceMap'. These properties are valid:
   object { test?, include?, exclude?, terserOptions?, extractComments?, parallel?, minify? }
```

The release config has probably been removed? :)

Reply:
I downgraded Terser webpack plugin to v4 that seams to solved sthe problem